### ISSUE TITLE
tpot iso doesnt appear to install anything.

### ISSUE THREAD

**Issue by R3-d4ct3d** [dtag-dev-sec/tpotce#546]
I'm currently trying to install T-Pot 19.03.1 using the ISO available on Github but when I try to go through the installation process, it will install the base system and additional installs/select and install software but never prompts me for a username/pw combo and goes straight to finishing the install.  After that, it reboots back to the bios menu.  I've watched Youtube videos of others doing the same process and everything works fine for them so I have no idea what to do to fix it.  I have the system requirements to run T-Pot.  I'm also using Virtual Box. 

# Issues

Before you post your issue make sure it has not been answered yet and provide `basic support information` if you come to the conclusion it is a new issue:
- 🔍 Use the [search function](https://github.com/dtag-dev-sec/tpotce/issues?utf8=%E2%9C%93&q=) first
- 🧐 Check our [WIKI](https://github.com/dtag-dev-sec/tpotce/wiki)
- 📚 Consult the documentation of 💻 [Debian](https://www.debian.org/doc/), 🐳 [Docker](https://docs.docker.com/), the 🦌 [ELK stack](https://www.elastic.co/guide/index.html) and the 🍯 [T-Pot Readme](https://github.com/dtag-dev-sec/tpotce/blob/master/README.md).
- **⚠️ Provide [basic support information](#info) or similar information with regard to your issue or we can not help you and will close the issue without further notice**

<br>
<br>
<br>

<a name="info"></a>
## ⚠️ Basic support information (commands are expected to run as `root`)

- What version of the OS are you currently using `lsb_release -a` and `uname -a`?
Debian 10(version packaged with T-Pot 19.03.1)
- What T-Pot version are you currently using?
19.03.1
- What edition (Standard, Nextgen, etc.) of T-Pot are you running?
Don't get that far
- What architecture are you running on (i.e. hardware, cloud, VM, etc.)?
Virtual Box
- Did you have any problems during the install? If yes, please attach `/install.log` `/install.err`.
don't get any errors and haven't found anything in the logs to suggest an issue
- How long has your installation been running?
Haven't gotten it working yet
- Did you install upgrades, packages or use the update script?
no
- Did you modify any scripts or configs? If yes, please attach the changes.
no
- Please provide a screenshot of `glances` and `htop`.
doesn't fully install just reboots back to bios menu
- How much free disk space is available (`df -h`)?
doesn't fully install just reboots back to bios menu
- What is the current container status (`dps.sh`)?
doesn't fully install just reboots back to bios menu
- What is the status of the T-Pot service (`systemctl status tpot`)?
doesn't fully install just reboots back to bios menu
- What ports are being occupied? Stop T-Pot `systemctl stop tpot` and run `netstat -tulpen`
doesn't fully install just reboots back to bios menu
- If a single container shows as `DOWN` you can run `docker logs <container-name>` for the latest log entries
doesn't fully install just reboots back to bios menu

**Comment by t3chn0m4g3** [comment]
Cannot reproduce. Did you remove the virtual CD drive after installation?

**Comment by Fosocles** [comment]
Are you using tpot.iso or the install script with the --type=iso argument? If the latter, then you should use the already packed .iso file.

**Comment by R3-d4ct3d** [comment]
So i'm using the tpot.iso file that you guys have up on github.  I pulled that down and then spun up a Debian instance with the system requirements.  I load the iso file into the optical drive and start the vm.  It takes me to the bios menu and I run through the installation process but once all the installation is 'done' and it reboots, it just takes me back to the bios screen.  I've tried unmounting the iso file and then starting it back up but it just takes me back to the bios screen and wants me to do the same thing. But after reboot, it again goes to the bios screen.  I've been trying to get it to work for at least a day so I could be overlooking something small at this point but I can't figure it out.
